platform/ext/target/nordic_nrf/common/gcc/nordic_nrf_s.ld.template

```diff
@@ -0,0 +1,691 @@
+;/*
```

Comment:
Do we actually need this file?

Reply:
Not sure..

Reply:
Let's keep it until we're sure?

Reply:
It does not seem to be used anywhere.

Reply:
`tfm_common_s.ld.template` is referred here: https://github.com/NordicPlayground/trusted-firmware-m/blob/2ac4b29ad8a8f2d5697b1ee3b5157077402690d2/tools/tfm_generated_file_list.yaml#L39-L40

Reply:
It can be used with a script to regenerate the linker script. Maybe it's needed if we add custom secure services. I haven't fully understood when it's used.

Reply:
ok, so let's leave it

Reply:
Maybe we should add our file to that list?

Reply:
Yes, let's keep this file for now and add it to that list later, if needed (or drop, otherwise). But it seems that the contents of this file should be updated (it doesn't have the veneers stuff defined as a macro).

Reply:
Thanks, fixed.